Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/AnalyticsController.cs

```diff
@@ -19,6 +18,7 @@ public AnalyticsController(IAnalyticsService analyticsService, AnalyticsConfigur
             Configuration = configuration;
 
             analytics.AddPlugin(new StaticCommonTraitsPlugin(launcherTraits));
+            analytics.Identify(SystemInfo.deviceUniqueIdentifier);
 
             analytics.Track(AnalyticsEvents.General.SYSTEM_INFO_REPORT, new JsonObject
```

Comment:
since once I saw that SYSTEM_INFO didn't arrive, let's put a Flush directly after first `Identify`.